test/e2e_flags_test.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"flag"
+	"fmt"
+	"testing"
+)
+
+func TestE2eFlags(t *testing.T) {
+	v := flag.Lookup("v")
+	if v == nil {
+		t.Fatal("Could not find 'v' flag; klog was not init-ed")
+	}
+	if v.Value.String() != "2" {
+		t.Fatalf("Either '%s' was passed in for v or v is not being initialized properly. The former is a test limitation, but the latter is an actual problem", v.Value.String())
+	}
+	if flagsSetup {
+		t.Error("SetupLoggingFlags should not have been called by the unit tests")
+	}
+
+	var flagTests = []struct {
+		log         string
+		logverbose  bool
+		nonDefaultV string
+		expectedV   string
+	}{
+		{"Check default case with logverbose off", false, "", "2"},
```

Comment:
nit: I find it easier to read with the values on their own lines and prefixed with the type.

Reply:
I don't know what "prefixed with the type" means, but they're on separate lines

Reply:
```
{
   name: "Check default case with logverbose off",
   logVerbose: false,
   expectedV: "2",
},
{
   name: "Check ..."
   ...
 }
```

Reply:
done